lib/state/action-types.ts

```diff
@@ -55,6 +55,10 @@ export type CreateNote = Action<'CREATE_NOTE'>;
 export type FilterNotes = Action<'FILTER_NOTES', { notes: T.NoteEntity[] }>;
 export type Search = Action<'SEARCH', { searchQuery: string }>;
 export type SetAuth = Action<'AUTH_SET', { status: AuthState }>;
+
+export type SetSearchFocus = Action<'SET_SEARCH_FOCUS'>;
+export type UnsetSearchFocus = Action<'UNSET_SEARCH_FOCUS'>;
```

Comment:
I like this change but `SetSearchFocus` and `UnsetSeachFocus` read a bit awkward to me. Did you have thoughts about making it active such as with `FocusSearch` and `UnfocusSearch` or `BlurSearch` (if we wanted to stick with HTML terminology)

Reply:
Fine by me, I will update it